src/ncp/ncp_base_ftd.cpp

```diff
@@ -57,6 +57,39 @@
 namespace ot {
 namespace Ncp {
 
+void NcpBase::HandleParentResponseInfo(otThreadParentResponseInfo *aInfo, void *aContext)
```

Comment:
Can you please move these method to this existing section in the `ncp_base_ftd` file.
```
// ----------------------------------------------------------------------------
// MARK: Property/Status Changed
// ----------------------------------------------------------------------------
```

Reply:
sure